nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ListenRELP.java

```diff
@@ -127,67 +191,48 @@ public void onScheduled(ProcessContext context) throws IOException {
         return results;
     }
 
-    @Override
-    protected ChannelDispatcher createDispatcher(final ProcessContext context, final BlockingQueue<RELPEvent> events) throws IOException {
-        final EventFactory<RELPEvent> eventFactory = new RELPEventFactory();
-        final ChannelHandlerFactory<RELPEvent,AsyncChannelDispatcher> handlerFactory = new RELPSocketChannelHandlerFactory<>();
+    private void initializeRELPServer(final ProcessContext context) throws IOException {
```

Comment:
Recommend using camel-case for method naming:
```suggestion
    private void initializeRelpServer(final ProcessContext context) throws IOException {
```